### PR TITLE
update project structure and use latest apis

### PR DESCRIPTION
**Pull request by kshitij12345**
Update the run structure and use latest API's

NOTE: This PR is not for merging, just for review.

### CODE REVIEW

**Comment by kshitij12345** [comment]
@SiddhantSadangi can you please do a review to see if I missed something? Thank you :)

File: ci_evaluate.py [hunk 1]
```diff
@@ -6,8 +6,10 @@
 from dataset import BrainSegmentationDataset
 from model_utils import DiceLoss, UNet
 
+os.environ["NEPTUNE_PROJECT"] = "common/project-images-segmentation-update"
```
Comment (SiddhantSadangi) [inline ci_evaluate.py:9]:
```suggestion
os.environ["NEPTUNE_PROJECT"] = "common/project-images-segmentation"
```

File: ci_monitoring.py [hunk 2]
```diff
@@ -3,23 +3,18 @@
 
 import neptune.new as neptune
 
+os.environ["NEPTUNE_PROJECT"] = "common/project-images-segmentation-update"
```
Comment (SiddhantSadangi) [inline ci_monitoring.py]:
```suggestion
os.environ["NEPTUNE_PROJECT"] = "common/project-images-segmentation"
```

File: finetune.py [hunk 3]
```diff
@@ -68,8 +68,11 @@ def main(args):
     # Fetch Previous Best Run for Finetuning #
     ##########################################
 
+    # (neptune) set project name
+    os.environ["NEPTUNE_PROJECT"] = "common/project-images-segmentation-update"
```
Comment (SiddhantSadangi) [inline finetune.py:72]:
```suggestion
    os.environ["NEPTUNE_PROJECT"] = "common/project-images-segmentation"
```

File: notebooks/FetchRunMetadata.ipynb [hunk 4]
```diff
@@ -38,7 +38,7 @@
     }
    ],
    "source": [
-    "project = neptune.init_project(name=\"common/project-images-segmentation\")\n",
+    "project = neptune.init_project()\n",
```
Comment (SiddhantSadangi) [inline notebooks/FetchRunMetadata.ipynb]: `NEPTUNE_PROJECT` hasn't been set

File: notebooks/ProjectMetaData.ipynb [hunk 5]
```diff
@@ -22,7 +22,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "project = neptune.init_project(name=\"common/project-images-segmentation\")\n",
+    "project = neptune.init_project()\n",
```
Comment (SiddhantSadangi) [inline notebooks/ProjectMetaData.ipynb]: `NEPTUNE_PROJECT` hasn't been set

File: notebooks/ProjectMetaData.ipynb [hunk 6]
```diff
@@ -57,28 +57,56 @@
...
+      "https://app.neptune.ai/common/project-images-segmentation-update/m/IMG-MOD\n",
+      "Remember to stop your model once you’ve finished logging your metadata (https://docs.neptune.ai/api/model#stop). It will be stopped automatically only when the notebook kernel/interactive console is terminated.\n"
+     ]
+    }
+   ],
    "source": [
     "import neptune.new as neptune\n",
     "\n",
-    "model = neptune.init_model(name=\"Prediction model\", key=\"MOD\", project=\"common/project-images-segmentation\")"
+    "model = neptune.init_model(name=\"Prediction Model\", key=\"MOD\", project=\"common/project-images-segmentation-update\")"
```
Comment (SiddhantSadangi) [inline notebooks/ProjectMetaData.ipynb]:
```suggestion
    "model = neptune.init_model(name=\"Prediction Model\", key=\"MOD\")"
```

File: train.py [hunk 7]
```diff
@@ -283,9 +289,7 @@ def main(args):
         best_run["sys/tags"].remove("best")
 
         # (neptune) add current model as a new version in model registry.
-        model_version = neptune.init_model_version(
-            model="IMGSEG-MOD", project="common/project-images-segmentation"
-        )
+        model_version = neptune.init_model_version(model="IMG-MOD")
```
Comment (SiddhantSadangi) [inline train.py:292]:
```suggestion
        model_version = neptune.init_model_version(model="IMGSEG-MOD")
```

File: train.py [hunk 8]
```diff
@@ -304,7 +308,7 @@ def main(args):
     parser.add_argument(
         "--epochs",
         type=int,
-        default=25,
+        default=10,
         help="number of epochs to train (default: 25)",
```
Comment (SiddhantSadangi) [inline train.py]:
```suggestion
        help="number of epochs to train (default: 10)",
```

**Review by SiddhantSadangi — CHANGES_REQUESTED** [review]
Can you run this e2e in the original [project-images-segmentation](https://app.neptune.ai/o/common/org/project-images-segmentation) and update the `Overview` dashboard with the changes?

**Comment by kshitij12345** [comment]
Ah, sorry I forgot to mention this is just to verify correct use of new API, I don't intend to merge this.

We are logging to `project-images-segmentation-update` for testing and iterating.

File: notebooks/ProjectMetaData.ipynb (hunk 5 above)
Comment (kshitij12345) [reply notebooks/ProjectMetaData.ipynb]: Thanks!

File: notebooks/FetchRunMetadata.ipynb (hunk 4 above)
Comment (kshitij12345) [reply notebooks/FetchRunMetadata.ipynb]: Thanks!

File: train.py [hunk 9]
```diff
@@ -94,10 +94,10 @@ def main(args):
 
         if outline_image.max() > 1:
             outline_image = outline_image.astype(np.float32) / 255
-        # (neptune) Log sample images with mask overlay
+        # (Neptune) Log sample images with mask overlay
         run["data/samples/images"].log(File.as_image(outline_image), name=fname)
```
Comment (SiddhantSadangi) [inline train.py]:
```suggestion
        run["data/samples/images"].append(File.as_image(outline_image), name=fname)
```

File: train.py [hunk 10]
```diff
@@ -166,7 +173,7 @@ def main(args):
             loss.backward()
             optimizer.step()
 
-            # (neptune) Log train loss after every step
+            # (Neptune) Log train loss after every step
             run["training/metrics/train_dice_loss"].log(loss.item())
```
Comment (SiddhantSadangi) [inline train.py]:
```suggestion
            run["training/metrics/train_dice_loss"].append(loss.item())
```

File: train.py [hunk 11]
```diff
@@ -191,7 +198,7 @@ def main(args):
                 y_pred = unet(x)
                 loss = dsc_loss(y_pred, y_true)
 
-                # (neptune) Log valid loss after every step
+                # (Neptune) Log valid loss after every step
                 run["training/metrics/validation_dice_loss"].log(loss.item())
```
Comment (SiddhantSadangi) [inline train.py]:
```suggestion
                run["training/metrics/validation_dice_loss"].append(loss.item())
```

File: train.py [hunk 12]
```diff
@@ -226,7 +233,7 @@ def main(args):
                                 desc = (
                                     f"Epoch: {epoch}\nPatient: {patient_name}\nImage No: {img_no}"
                                 )
-                                # (neptune) Log prediction and ground-truth on original image
+                                # (Neptune) Log prediction and ground-truth on original image
                                 run[f"training/validation_prediction_progression/{fname}"].log(
```
Comment (SiddhantSadangi) [inline train.py]:
```suggestion
                                run[f"training/validation_prediction_progression/{fname}"].append(
```

File: train.py [hunk 13]
```diff
@@ -254,42 +261,47 @@ def main(args):
             # corresponding dice coefficient
```
Comment (SiddhantSadangi) [inline train.py:265]: `run["training/metrics/validation_dice_coefficient"].append(mean_dsc)`

File: train.py [hunk 14]
```diff
@@ -127,10 +134,10 @@ def main(args):
     model_vis = make_dot(y.mean(), params=dict(unet.named_parameters()))
     model_vis.format = "png"
     model_vis.render("model_vis")
```
Comment (SiddhantSadangi) [inline train.py]:
```suggestion
```
we don't need these anymore

**Review by SiddhantSadangi — CHANGES_REQUESTED** [review]
Just a few leftover `log()` methods

**Pull request closed without merging**